docs/1-introduction/1-1-getting-started/1-1-0-overview.md

```diff
@@ -1,23 +1,31 @@
 # Overview
 
-As many may know or not know, there's a few choices in terms of what to use for graphics, such as OpenGL, Vulkan, Metal and what is commonly known as "DirectX".
-However in word of mouth DirectX usually refers to Direct3D, which is one of the many API's that DirectX has, as it is less known that DirectX is more than just graphics, it is:
+As you may or may not know, there's a few choices in terms of what you can use for rendering, such
+as OpenGL, Vulkan, Metal, and what is commonly known as "DirectX". However, in word of mouth
+DirectX usually refers to Direct3D, which is one of the many API's that DirectX has. In fact
+DirectX provides tooling and libraries for more aspects of game development, including:
 
 - **Audio**
-
 - **Fonts**
-
 - **Input**
-
 - **Graphics**
 
-And _other things_ more or less targeted at stuff one needs for game development.
+The goal of LearnD3D11 is, as one might guess, the targeted explanation and showcase of Direct3D11
+within C++, or as one generally refers to it: "DirectX 11". This guide does _NOT_ cover DirectX 12,
+the newer version of the API, as it is very low level and will usually require a good understanding
+of the GPU and how it works. We will cover concepts and some techniques used in graphics programming,
+and we do not expect you to have any prerequisites in this field.  It is, however, not a guide
+covering C++ or programming in general; we expect at least the ability and understanding to write
+object-oriented programs in C++.
 
-The goal of LearnD3D11 is as one might guess is the targeted explanation and showcase of Direct3D11 or as one generally refers to it: "DirectX 11".
+During each step we'll provide a project for you to follow along as we explain everything for you
+to start rendering geometry using your GPU.
 
-During each step we'll provide a project for you to follow along as we explain each individual thing needed to get some graphics on your own screen.
+Also note that DirectX is made by Microsoft and is generally only available on Windows. However,
+for DirectX 12 translation layers such as vkd3d exist for it to work on Linux as well.
```

Comment:
there is also `DXVK` if you want to mention that, vkd3d is probably a lesser known translation layer?